src/Toolkit/I18n.php

```diff
@@ -190,6 +198,20 @@ public static function translations(): array
         return static::$translations;
     }
 
+    /**
+     * Returns (and creates) a decimal number formatter for a given locale
+     *
+     * @return array
+     */
+    protected static function decimalNumberFormatter(string $locale): NumberFormatter
+    {
+        if (isset(static::$decimalNumberFormatters[$locale])) {
+            return static::$decimalNumberFormatters[$locale];
+        }
+
+        return static::$decimalNumberFormatters[$locale] = new NumberFormatter($locale, NumberFormatter::DECIMAL);
```

Comment:
There should be a check for the existence of the `NumberFormatter` class here. If the class is not defined (= if the `Intl` extension is not loaded), `null` should be returned.